### DISCUSSION TITLE
Trying to run nowcasting-app locally

### DISCUSSION THREAD

**Discussion by ram-from-tvl** [openclimatefix/quartz-frontend#529] [Q&A]
Can someone help me with this error i am getting when i try to run nowcasting-app locally on my Vscode in Ubuntu 24 LTS. i am attaching a screenshot for reference..?
![Screenshot from 2024-07-18 02-55-20](https://github.com/user-attachments/assets/be593c6c-0ebf-4275-bd79-d41438d26a9e)

**Comment by braddf** [comment]
Hey @ram-from-tvl 👋 

Could you give a little more detail on steps to reproduce, i.e. assuming this is a fresh install, what commands you have run after `git clone` etc. and will see if can find the issue

>FYI we have an ongoing issue with Auth for contributors to be able to connect to our API properly from the frontend apps, but this should be sorted within the next week, but this shouldn't be causing your error – it should run the app, but potentially fail on the Auth part of the dev build and say something like `AUTH0_SECRET is required`

**Comment by suvanbanerjee** [comment]
@braddf today I tried running the app, the auth0 is giving issue when running locally it redirects here 
![image](https://github.com/user-attachments/assets/4561bda5-c436-4d51-82c5-814b768ba545)

I have put the env as suggested in README but dont know how to fetch the client id and client secret

**Comment by suvanbanerjee** [comment]
I also tried the solution in https://github.com/mduffin95/quartz-frontend/pull/1/files but doing so I am getting fetch error 
![image](https://github.com/user-attachments/assets/7af6a82d-1a16-48db-9ae2-3e023ec1b14c)
